resources/language/resource.language.ro_ro/strings.po

```diff
@@ -84,7 +84,7 @@ msgstr "Alte opțiuni"
 
 msgctxt "#30016"
 msgid "If you log out, all account data will be deleted, but the exported video files in the Kodi library will be preserved.[CR]Do you want to continue?"
-msgstr ""
+msgstr "Dacă vă deconectați, datele contului vor fi șterse, dar fișierele video exportate în librăria Kodi vor fi păstrate.[CR]Doriți să continuați"
```

Comment:
the lack of "?" in the question is intentional?

Reply:
I forgot about the question mark. I rectified it in a commit.